src/interface.pre

```diff
@@ -10,6 +10,10 @@ BEGIN:
         P11PROV_raise(ctx, ret, "Can't get module interfaces"); \
         return ret; \
     } \
+    if (p11prov_ctx_is_call_blocked(ctx, P11PROV_BLOCK_##name)) { \
+        P11PROV_raise(ctx, ret, "C_" #name " is blocked"); \
```

Comment:
This is super-ugly in the generated code. Can we do something like even though it would cause some more copying in runtime in theory?
```suggestion
        P11PROV_raise(ctx, ret, "C_%s is blocked", #name); \
```
(I would hope the compiler can optimize these though)

Reply:
it probably won't, but I am also changing this raise to a debug, as one of the reasons we had to block the operation was also unnecessary noise in the openssl logs